lib/Doctrine/ORM/Persisters/SqlValueVisitor.php

```diff
@@ -84,7 +84,7 @@ public function walkValue(Value $value)
      *
      * @return mixed[][]
      *
-     * @psalm-return array{0: array, 1: array}
+     * @psalm-return array{0: array, 1: array<array>}
```

Comment:
Let's use `mixed[]` here and there?

Reply:
replaced by array<mixed> to keep coherence